src/views/acquirer/format.njk

```diff
@@ -13,7 +13,8 @@
             id: "something-else",
             name: "something-else",
             classes: "govuk-!-width-two-thirds",
-            value: savedValue['something']['explanation']
+            value: savedValue['something']['explanation'],
+            errorMessage: errorMessage
```

Comment:
not tested but shouldnt this look like `errorMessage['something-else']`
I only ask because thats how the other templates access the field error